src/main.ts

```diff
@@ -62,6 +75,12 @@ const command: Command = {
 			default: false
 		});
 
+		options('i', {
```

Comment:
So we feel that changing the targetted environments is to be supported.  What about testing against safari or firefox locally?  It is basically Chrome or `eject`?

If that is the case, why do we even offer `.dojorc` if it is our opinionated way or `eject`?  I would be comfortable allowing some sort of twiddling of common intern configuration settings (like environments, passing the username/password for the tunnel, changing environments, etc.).